src/app/client/src/app/modules/observation/components/entity-list/entity-list.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit, Input, EventEmitter, Output } from '@angular/core';
+@Component({
+    selector: 'entity-list',
+    templateUrl: './entity-list.component.html',
+    styleUrls: ['./entity-list.component.scss']
+})
+export class EntityListComponent implements OnInit {
+    @Input() entities;
+    @Input() selectedEntity;
+    @Output() onDelete = new EventEmitter();
```

Comment:
we should have only one  event as output,, use different actions to differentiate between outputs
eg output() onAction

for delete this.onAction.emit({action:'delete', data: {id:""}})